### PR TITLE
Folder restructure for Search API SDKs [ EntitySearch,WebSearch,ImageSearch,VideoSearch,SpellCheck,CustomSearch ]

### PR DESCRIPTION
**Pull request by sudhansu88**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ ] Please add REST spec PR link to the SDK PR
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
Overall looks good, but looking at the directory names, make sure you are not getting into MAX_PATH issues during build.
Also I have seen you have not pulled upstream changes to these project files.
Almost all project files need the following header import and footer imports as you see in this file
https://github.com/sudhansu88/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Management.Compute/Microsoft.Azure.Management.Compute.csproj

Also make sure you don't explicitly add any directories or files to the sdk project.
I see couple of places you have added Folder Include="Properties" (an additional ItemGroup). Please remove any ItemGroup that you don't need.

File: src/SDKs/CognitiveServices/dataPlane/Search/BingWebSearch/BingWebSearch/Microsoft.Azure.CognitiveServices.Search.WebSearch.csproj [hunk 1]
```diff
@@ -1,5 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
   <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code below this line -->
```
Comment (shahabhijeet) [inline src/SDKs/CognitiveServices/dataPlane/Search/BingWebSearch/BingWebSearch/Microsoft.Azure.CognitiveServices.Search.WebSearch.csproj]: @sudhansu88 this should be 
Please do not move/edit code above this line and it says "below"
Can you make the change in all the csproj files.

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
Minor changes required. Once done this PR is good to be merged.

**Comment by sudhansu88** [comment]
Updated Comment as per suggestion

**Comment by brjohnstmsft** [comment]
@sudhansu88 Please include the specific name of the service in your PR titles. There is more than one "Search API/SDK" in this repo. Thanks!

**Comment by sudhansu88** [comment]
@brjohnstmsft : Updated all SDK names. Please merge.

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**